### DISCUSSION TITLE
E external/xla/xla/stream_executor/cuda/cuda_dnn.cc:407] There was an error before creating cudnn handle (302): cudaGetErrorName symbol not found. : cudaGetErrorString symbol not found.

### DISCUSSION THREAD

**Discussion by TRo202** [jax-ml/jax#21712] [Q&A]
Hi.

Previously I had no problem using JAX on my PC, that has CUDA version of 11.

Now, I'm trying to use JAX on my laptop, whose CUDA version is 12.2.  

So I installed JAX with the code below
`pip install --upgrade "jax[cuda12_pip]" -f https://storage.googleapis.com/jax-releases/jax_cuda_releases.html`

It was installed successfully, but when I used JAX, I got the following error:
```
2024-06-07 03:14:00.848518: E external/xla/xla/stream_executor/cuda/cuda_dnn.cc:407] There was an error before creating cudnn handle (302): cudaGetErrorName symbol not found. : cudaGetErrorString symbol not found
...
jax._src.traceback_util.UnfilteredStackTrace: jaxlib.xla_extension.XlaRuntimeError: FAILED_PRECONDITION: DNN library initialization failed. Look at the errors above for more details.
```

But when I checked
```
from jax.lib import xla_bridge
print(xla_bridge.get_backend().platform)
```
I did get `gpu`

I uninstalled JAX and JAXlib again and installed again with 
`pip install "jax[cuda]"==0.4.13 -f https://storage.googleapis.com/jax-releases/jax_cuda_releases.html`
but I still got the same error.

I tried to check the cuDNN version
`cat /usr/local/cuda/include/cudnn_version.h | grep CUDNN_MAJOR -A 2
`
but I got 
`cat: /usr/local/cuda/include/cudnn_version.h: No such file or directory
`
So I printed cuDNN version manually and I got
```
>>> import torch
>>> print("cuDNN version:", torch.backends.cudnn.version())
cuDNN version: 8902
```

CUDA version, cuDNN version, and JAX version seem to have no problem, but I don't know why I'm getting this error.

I've been struggling with this for hours, and I would greatly appreciate if someone could help me with this. Thanks in advance.

**Comment by prateekiiest** [comment]
Same here

**Reply by TRo202** [reply]
Since I couldn't get any help regarding this issue, I just had to install the lower version (11)...

**Comment by Topdp** [comment]
Have you solved it now? This question has been bothering me for many days.

**Reply by TRo202** [reply]
No... I just installed the lower version of CUDA (11.3)

**Reply by yht968** [reply]
same trouble :(
